tests/conftest.py

```diff
@@ -33,7 +33,11 @@ def pytest_configure(config):
 def get_tests_daft_runner_name() -> Literal["ray"] | Literal["native"]:
     """Test utility that checks the environment variable for the runner that is being used for the test."""
     name = os.getenv("DAFT_RUNNER")
-    assert name is not None, "Tests must be run with $DAFT_RUNNER env var"
+    if name is None:
+        import warnings
+
+        warnings.warn("DAFT_RUNNER environment variable is not set, using default value 'native'")
+        name = "native"
```

Comment:
A more descriptive error message does make sense but I think we consciously did not set a default because when we are testing or benchmarking, we actually want people to be explicit about what runner they are using. 

What about this instead?


```suggestion
    if name is None:
        raise ValueError("Daft Python tests require the `DAFT_RUNNER` environment variable to be explicitly set. Please run with `DAFT_RUNNER=native` or `DAFT_RUNNER=ray`.")
```

Same for the benchmark script.